www/src/components/navigation/leftSidebar.astro

```diff
@@ -27,14 +27,14 @@ const sidebar = SIDEBAR[langCode];
   </div>
   <ul
     id="nav-groups"
-    class="p-4 md:p-8 h-full overflow-x-visible overflow-y-scroll t3-scrollbar sm:overflow-auto pb-32 dark:text-t3-purple-50 text-slate-900"
+    class="p-4 md:p-8 h-screen overflow-x-visible overflow-y-scroll t3-scrollbar sm:overflow-auto pb-32 dark:text-t3-purple-50 text-slate-900"
```

Comment:
should this be screen height? can we not just `flex-1` it or smth so that it spans down to the footer?